src/richie/apps/courses/models/course.py

```diff
@@ -132,6 +132,10 @@ def get_course_runs(self):
         """
         Returns a query yielding the course runs related to the course. They may be direct
         children of the course or children of a snapshot of the course.
+
+        For a draft course, the related draft course runs are retrieved.
+        For a public course, the related public course runs are retrieved, but only if they
```

Comment:
The tests seem to cover all cases, but I just don't understand the explanation clearly here: what is a `public` course run if not a `published` course run?

Reply:
the public course run is the instance representing the published course run as opposed to the draft one.

By published I mean that it needs to be currently publicly visible.

The catch is that a page in DjangoCMS can be unpublished... The public object is created the first time a draft object is published. But then of course this `public` instance will still exist if the object is then unpublished (in all languages)... so a page can have a public version of itself in database but not be `published` in any language...  
I'm going to try to rephrase this docstring.

Reply:
Ok thanks for the clarification, I'm not as familiar with Django CMS so it was not obvious to me 👍